### PR TITLE
PIL 870 - Accessibility title changes

### PR DESCRIPTION
**Pull request by MarjanMubarok**

### CODE REVIEW

File: test/controllers/subscription/manageAccount/ManageGroupDetailCheckYourAnswersControllerSpec.scala [hunk 1]
```diff
@@ -74,7 +74,7 @@ class ManageGroupDetailCheckYourAnswersControllerSpec extends SpecBase with Summ
         val request = FakeRequest(GET, controllers.subscription.manageAccount.routes.ManageGroupDetailsCheckYourAnswersController.onPageLoad.url)
         val result  = route(application, request).value
         status(result) mustEqual OK
-        contentAsString(result) must include("Check your answer")
+        contentAsString(result) must include("Group details")
```
Comment (AKKUM) [inline test/controllers/subscription/manageAccount/ManageGroupDetailCheckYourAnswersControllerSpec.scala:77]: Can we do full content test. like contentAsString(result) mustEqual view()

File: test/controllers/subscription/manageAccount/ManageGroupDetailCheckYourAnswersControllerSpec.scala [hunk 2]
```diff
@@ -92,7 +92,7 @@ class ManageGroupDetailCheckYourAnswersControllerSpec extends SpecBase with Summ
...
```
Comment (AKKUM) [inline test/controllers/subscription/manageAccount/ManageGroupDetailCheckYourAnswersControllerSpec.scala:95]: Same here

File: test/controllers/subscription/manageAccount/ManageGroupDetailCheckYourAnswersControllerSpec.scala (hunk 1 above)
Comment (AKKUM) [reply test/controllers/subscription/manageAccount/ManageGroupDetailCheckYourAnswersControllerSpec.scala:77]: Every other check your Answer Controller is not tested with full content. So approving it.

**Review by jassalrichy — APPROVED** [review]

File: conf/messages.en [hunk 3]
```diff
@@ -679,6 +679,8 @@ contactCheckYourAnswers.heading.caption = Contact details
 contactCheckYourAnswers.second = Second contact
 contactCheckYourAnswers.address = Contact address
 
+manageContactCheckYourAnswers.title = Contact details
```
Comment (AKKUM) [inline conf/messages.en:682]: Should it not be contactCheckYourAnswer.title ?

Comment (MarjanMubarok) [reply conf/messages.en:682]: There are two different pages which previously shared the same message key, so I changed the name so it is specific to the manage contact details page.

Comment (MarjanMubarok) [reply conf/messages.en:682]: The pages now how their own separate set of keys. The <br> tag has been removed as well. As far as I know the <br> tag isn't doing anything.

**Review by AKKUM — APPROVED** [review]

**Pull request merged**